restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/AccountsMigrationEndpointTest.java

```diff
@@ -0,0 +1,148 @@
+        String newOrganizationSid = "ORafbe225ad37541eba518a74248f0ac4d";
+
+        JsonObject currentParentAccout = RestcommAccountsTool.getInstance().getAccount(deploymentUrl.toString(), adminUsername, adminAuthToken, parentSid);
+        assertNotNull(currentParentAccout);
+        String currentOrg = currentParentAccout.get("organization").getAsString();
+        assertEquals(originalOrganizationSid, currentOrg);
+
+        JsonObject currentChildAccout = RestcommAccountsTool.getInstance().getAccount(deploymentUrl.toString(), adminUsername, adminAuthToken, childSid1);
+        assertNotNull(currentChildAccout);
+        String currentChildOrg = currentChildAccout.get("organization").getAsString();
+        assertEquals(originalOrganizationSid, currentChildOrg);
+
+        //Migrate Parent Account and child accounts
+        RestcommAccountsTool.getInstance().migrateAccount(deploymentUrl.toString(), adminUsername,
+                adminAuthToken, parentSid, newOrganizationSid);
+
+        JsonObject migratedParentAccount = RestcommAccountsTool.getInstance().getAccount(deploymentUrl.toString(), adminUsername, adminAuthToken, parentSid);
+        assertNotNull(migratedParentAccount);
+        String parentNewOrg = migratedParentAccount.getAsJsonObject().get("organization").getAsString();
+        assertEquals(newOrganizationSid, parentNewOrg);
+        assertTrue(!currentOrg.equalsIgnoreCase(parentNewOrg));
+
+        JsonObject childAccount1 = RestcommAccountsTool.getInstance().getAccount(deploymentUrl.toString(), adminUsername, adminAuthToken, childSid1);
+        assertNotNull(childAccount1);
+        String childNewOrg = childAccount1.get("organization").getAsString();
+        assertEquals(newOrganizationSid, childNewOrg);
+
+        //Migrate Parent Account and child accounts
+        assertNotNull(RestcommAccountsTool.getInstance().migrateAccount(deploymentUrl.toString(), adminUsername,
+                adminAuthToken, parentSid, originalOrganizationSid));
+    }
+
+
+    @Test
+    public void testAttemptToMigrateAchildAccountShouldFail() {
+        String childSid1 = "AC55555555555555555555555555555555";
+
+        String originalOrganizationSid = "ORafbe225ad37541eba518a74248f0ac4c";
+        String newOrganizationSid = "ORafbe225ad37541eba518a74248f0ac4d";
+
+        JsonObject currentChildAccout = RestcommAccountsTool.getInstance().getAccount(deploymentUrl.toString(), adminUsername, adminAuthToken, childSid1);
+        assertNotNull(currentChildAccout);
+        String currentChildOrg = currentChildAccout.get("organization").getAsString();
+        assertEquals(originalOrganizationSid, currentChildOrg);
+
+        //Migrate Parent Account and child accounts
+        JsonObject response = RestcommAccountsTool.getInstance().migrateAccount(deploymentUrl.toString(), adminUsername,
+                adminAuthToken, childSid1, newOrganizationSid);
+
+        assertNull(response);
```

Comment:
can we assert status code as well?

Reply:
@jaimecasero I don't think its important.